src/main/java/org/openrewrite/javascript/tree/JS.java

```diff
@@ -403,6 +403,105 @@ public CoordinateBuilder.Statement getCoordinates() {
         }
     }
 
+    @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+    @EqualsAndHashCode(callSuper = false, onlyExplicitlyIncluded = true)
+    @AllArgsConstructor(access = AccessLevel.PRIVATE)
+    @Data
+    final class Binary implements JS, Expression, TypedTree {
```

Comment:
I think `JsBinary` should be renamed instead of adding a `Binary` class. The change will require an update in `moderne-ast-write` to handle existing `JsBinary` in old LSTs.

Alternatively, we can add a new issue and handle that later, while adding to the existing class.